workflows/text_manipulation/filter_hvg/test-data/input.tabular

```diff
@@ -0,0 +1,1000 @@
+gene	MB-2960	MB-0511	MB-5204
```

Comment:
Can you describe the header in the readme and description ? I assume this is mandatory ?